tip-wlan/charts/wlan-port-forwarding-gateway-service/templates/secret.yaml

```diff
@@ -19,6 +10,4 @@ metadata:
     heritage: {{ .Release.Service | quote }}
 type: Opaque
 data:
-  {{- if .Values.creds.enabled }}
-  websocketSessionTokenEncKey: {{ .Values.creds.websocketSessionTokenEncKey | b64enc | quote }}
-  {{- end }}
\ No newline at end of file
+  websocketSessionTokenEncKey: {{ .Values.creds.websocketSessionTokenEncKey | b64enc | quote }}
```

Comment:
Same comment here as before - why we remove the if clause?

Reply:
took this offline with max, agreed that this is fine for now